conf/defaults.ini

```diff
@@ -397,6 +397,7 @@ global_session = -1
 [alerting]
 # Makes it possible to turn off alert rule execution.
 execute_alerts = true
+show_alerting_ui = true
```

Comment:
I would prefer an option that completely disables alerting (hides from UI and disables alerting engine / scheduler). Is that not the point of the show_alerting_ui? don't like this option that only hides it from the UI but the engine is still running.